src/OneExplorer/OneExplorer.ts

```diff
@@ -137,7 +137,7 @@ enum NodeType {
   product,
 }
 
-abstract class Node {
+export abstract class Node {
```

Comment:
Q. Is this change required?

Reply:
It is necessary to distinguish whether the viewer was executed in ONE or in Explorer. it is classified as an instance of a node.  However, it may be classified in other ways. Should we not export as much as possible?

Reply:
> Should we not export as much as possible?

I just wondered why it is exported. It looks that you have found out the other way and excluded this change.